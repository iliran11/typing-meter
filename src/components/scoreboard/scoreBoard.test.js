import React from 'react';
import ScoreBoard from './scoreBoard';
import { shallow } from 'enzyme';

const props = {
  wpm: 23,
  correctTypedWords: 20,
  disabled: true
};
const wrapper = shallow(<ScoreBoard {...props} />);

describe('`ScoreBoard`', () => {
  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
