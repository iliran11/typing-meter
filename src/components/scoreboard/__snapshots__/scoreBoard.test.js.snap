// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`ScoreBoard\` renders properly 1`] = `
<div
  className="scoreboard-container"
>
  <Toolbar
    className="secondary-toolbar transitionable joyride-step-scoreboard"
    noGutter={false}
    style={
      Object {
        "backgroundColor": null,
        "height": null,
        "justifyContent": null,
        "padding": null,
      }
    }
    text="TITLE"
  >
    <ToolbarGroup
      firstChild={false}
      lastChild={false}
    >
      <ScoreSection
        className="joyride-step--wpm"
        disabled={true}
        iconClass="fas fa-tachometer-alt"
        score={23}
        title="Words Per Minute"
      />
    </ToolbarGroup>
  </Toolbar>
</div>
`;
